app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt

```diff
@@ -3,12 +3,20 @@ package com.theopensourcefamily.chessclock
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import com.jakewharton.rxbinding3.view.clicks
+import dagger.Component
 import io.reactivex.Observable
 import kotlinx.android.synthetic.main.activity_clocks.*
 
+@Component
+interface ClockPresenterFactory{
```

Comment:
The linter here complains about the missing space:
```suggestion
interface ClockPresenterFactory {
```
A better place for this interface would be it's own file, outside the activity file.